src/App.js

```diff
@@ -1,24 +1,38 @@
-import React, {useEffect, useState} from "react";
+import React, {useEffect, useRef, useState} from "react";
 import FlashcardList from "./FlashcardList";
 import FlashcardData from "./data/flashcards.json";
+import LanguageData from "./data/languages.json"
+//import axios from "request";
 //import 'bootstrap/dist/css/bootstrap.min.css';
 
+
 function App() {
-  const [flashcards] = useState(FlashcardData)
-  const [languages, setLanguages] = useState(FlashcardData)
+  const [flashcards,setFlashcards] = useState(FlashcardData)
+  const [languages, setLanguages] = useState(LanguageData)
+
+    const languageEl = useRef(LanguageData)
+    const amountEl = useRef(FlashcardData)
+
+    const CardShuffler = (arr) => {
+        for (let i = arr.length - 1; i > 0; i--) {
```

Comment:
Use the lambda functions for array manipulation; Best way to shuffle an array: `array.sort( () => .5 - Math.random() );`